paper-input-behavior.html

```diff
@@ -224,6 +224,13 @@
       },
 
       /**
+       * Bind this to the `<input is="iron-input">`'s `oninvalid` property.
+       */
+      oninvalid: {
+        type: String
+      },
+
+      /**
        * Limits the numeric or date-time increments.
        * Bind this to the `<input is="iron-input">`'s `step` property.
        */
```

Comment:
`onInvalid` is not a property, it's an event listener for the native `input's` `invalid` event: https://developer.mozilla.org/en-US/docs/Web/Events/invalid. It's not fired every time the validity is checked, but whenever `checkValidity` is called.

Here's a Jsbin explaining this: http://jsbin.com/yonoho/edit?html,console,output

I think that if you want to listen to the `invalid` event, it's fair that you'd have to listen on the nested element (so `paperInput.inputElement`), since it's a non-bubbling event. However, maybe it would be better if `iron-input` called `checkValidity` in its validate() method, rather than just checking the `valid` property.